salt/modules/ssh.py

```diff
@@ -837,6 +853,33 @@ def get_known_host(user,
 
         salt '*' ssh.get_known_host <user> <hostname>
     '''
+    salt.utils.warn_until(
+            'Neon',
+            'get_known_host has been deprecated in favour of '
+            'get_known_host_entries.'
```

Comment:
Can you add a mention to this warning about when the function will be removed? Something like `\'get_known_host\' has been deprecated in favour of \'get_known_host_entries\'. \'get_known_host\' will be removed in Salt Neon.'`.

That will help users know when they need to upgrade by.